tools/docker/nginx/tests/run-tests.sh

```
@@ -0,0 +1,502 @@
+# Invoque via `sh run-tests.sh`, l'interpreteur est dash sur Debian et Ubuntu :
+# il ne connait ni `set -o pipefail` ni les tableaux associatifs. On se relance
+# donc sous bash. Ce test doit rester en syntaxe POSIX, c'est encore dash qui le lit.
+if [ -z "${BASH_VERSION:-}" ]; then
+  exec bash "$0" "$@"
+fi
+
+set -uo pipefail
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT="$(cd "$HERE/.." && pwd)"
+cd "$HERE"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)